src/main/kotlin/minesweeper/domain/Mine.kt

```diff
@@ -0,0 +1,19 @@
+package minesweeper.domain
+
+data class Mine(private val position: Pair<Int, Int>, val symbol: String = MINE_SYMBOL) {
+
+    fun getX(): Int = this.position.first
+    fun getY(): Int = this.position.second
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
    val x: Int = this.position.first
    val y: Int = this.position.second
```

Reply:
간단하고 좋아보이네요! 
그런데 한 가지 궁금한 점이 있습니다.
요구사항 중에 있는 내용인데요.

![image](https://user-images.githubusercontent.com/49539592/90462796-6cf14b80-e144-11ea-8d4f-0c7afed33703.png)

생성자의 프로퍼티까지도 인스턴스 변수로 보는 게 맞다고 생각하는데 그런가요?